### PR TITLE
PORTALS-2272

### PR DESCRIPTION
**Pull request by nickgros**
* Upgrade SRC to 3.0.35
* Rebuild lockfile
* fix new type errors
* Drop some unneeded dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -114,9 +113,11 @@
   },
   "resolutions": {
     "babel-jest": "^27.5.1",
+    "goober": "2.1.9",
     "js-yaml": "3.13.1",
     "plotly.js-basic-dist": "^1.47.3",
     "react": "16.14.0",
+    "react-hot-toast": "2.2.0",
```
Comment (nickgros) [inline package.json:120]: https://github.com/timolins/react-hot-toast/issues/170

File: public/index.html [hunk 2]
```diff
@@ -62,14 +62,10 @@
     <script src="https://cdn.jsdelivr.net/npm/markdown-it-container@2.0.0/dist/markdown-it-container.min.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/sanitize-html@1.20.0/dist/sanitize-html.min.js"></script>
     <script src="%PUBLIC_URL%/js/back-forward-nav-handler.js"></script>
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CEKFPZDZX7"></script>
-    <!-- include twitter widgets if using TwitterFeed component -->
     <script
       async
-      src="https://platform.twitter.com/widgets.js"
```
Comment (nickgros) [inline public/index.html:69]: No need for this anymore, right?

File: public/index.html [hunk 3]
```diff
@@ -62,14 +62,10 @@
     <script src="https://cdn.jsdelivr.net/npm/markdown-it-container@2.0.0/dist/markdown-it-container.min.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/sanitize-html@1.20.0/dist/sanitize-html.min.js"></script>
     <script src="%PUBLIC_URL%/js/back-forward-nav-handler.js"></script>
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-CEKFPZDZX7"></script>
```
Comment (nickgros) [inline public/index.html:65]: Just got reformatted below

File: src/portal-components/Versions.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import React, { useState, useEffect } from 'react'
 import moment from 'moment'
 import './Versions.css'
-import ReactTooltip from 'react-tooltip'
```
Comment (nickgros) [inline src/portal-components/Versions.tsx:4]: Only remaining reference to ReactTooltip

File: public/index.html (hunk 2 above)
Comment (jay-hodgson) [reply public/index.html:69]: Right!

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**